createTeamMasterK8s.groovy

```diff
@@ -167,6 +172,16 @@ ManagedMaster master = teamsFolder.createProject(ManagedMaster.class, teamName)
 master.setDisplayName(teamModel.getDisplayName())
 master.setDescription(masterDescription)
 
+/************************
+ * CONFIGURATION BUNDLE *
+ ************************/
+if(cascBundle?.trim()) {
+    // Note: ConnectedMasterCascProperty supported since 2.277.4.x. For earlier version, comment out the following.
```

Comment:
Probably also worth a comment that the following two lines are required to be in the strict order they are currently.

Reply:
You mean that the properties must be set in that specific order ? I didn't know about that. Adding it.

Reply:
@kyounger according to https://github.com/cloudbees/jenkins-scripts/pull/182, this should actually be token first, then the bundle.

Reply:
Yes, that matches what [works for me](https://github.com/kyounger/cbci-helmfile/blob/master/values/groovy/oc-custom-groovy/master-provisioner-shim.groovy#L58-L59) and what I was indicating earlier, but maybe wasn't fully clear on.